project/hca/extractors.py

```diff
@@ -3,7 +3,49 @@
 
 class FileExtractor:
     @staticmethod
-    def v1_0_0(metadata_file: dict) -> dict:
+    def v1_2(metadata_file: dict) -> dict:
+        # jmespath within the file objects
+        _format = "content.file_core.file_format || `null`"
```

Comment:
Consider inlining all these variables. There is no point in extracting them here. It would literally cut the LOCs in half.